### PR TITLE
Setting up the project

### PR DESCRIPTION
**Pull request by elemoine**
This PR sets up the Pyramid project. At this point this is just an empty, SQLAlchemy-based, Pyramid project, with an npm/node `package.json` file for managing the js and css libraries. The project also includes a `Makefile` for build tasks. See the README.rst file for more information.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: python
+
+python:
+- 2.7
+
+install:
+- make install
+
+script:
+- make check
```
Comment (ingenieroariel) [inline .travis.yml]: Should we add `make test` after this one right away?

**Comment by ingenieroariel** [comment]
Thanks for setting it up, I really like the Makefile, the fact that Travis is wired up from the start and the presence of flake8.

Travis was very quick to point out flake8 violations. Do you want to merge before or after fixing those?

```
thinkhazard/tests.py:10:9: N802 function name should be lowercase
thinkhazard/tests.py:24:9: N802 function name should be lowercase
thinkhazard/tests.py:37:9: N802 function name should be lowercase
thinkhazard/tests.py:41:9: F401 'Base' imported but unused
thinkhazard/tests.py:41:9: F401 'MyModel' imported but unused
thinkhazard/tests.py:47:9: N802 function name should be lowercase
thinkhazard/tests.py:55:47: W292 no newline at end of file
thinkhazard/views.py:17:80: E501 line too long (80 > 79 characters)
thinkhazard/views.py:36:1: W391 blank line at end of file
```

File: .travis.yml (hunk 1 above)
Comment (elemoine) [inline .travis.yml]: I think the `check` target should/will depend on the `test` target. So I'd just that `check` is the target we run on the CI system.

**Comment by elemoine** [comment]
@ingenieroariel I just fixed the flake8 errors.

**Comment by ingenieroariel** [comment]
Should we delete the elemoine_setup branch?

**Comment by elemoine** [comment]
> Should we delete the elemoine_setup branch?

I deleted it.

**Comment by elemoine** [comment]
@ingenieroariel Thanks for the review.

I like the idea of working with PRs and code reviews from the very beginning of the project. I'd also suggest that the PR author is the one merging the PR (after a review). The PR is indeed the one who usually knows if things are missing in the PR.

**Comment by ingenieroariel** [comment]
I agree with you @elemoine, having the author merging the PR makes a lot of sense.

**Pull request merged**